README.md

```diff
@@ -301,38 +327,41 @@ class Equations(Scene):
 ```
 Here, we will  look at very important concepts that will help when using Manim. 
 
-That looks long, but it's very simple. Here I have provided 2 ways of making equation and displaying it to the screen. If you remember, we installed some latex system requirements. We will use LaTex to make our equations look nice. 
+That looks long, but it's very simple. Here I have provided 2 ways of making equation and displaying it to the screen. If you remember, we installed some latex system requirements. We will use LaTeX to make our equations look nice. 
 
-LaTex will take it's own tutorial. However, you don't need to know a lot of LaTex. I will introduce some rules that will help you write any math equation. Notice that equations are specified in TextMobjects. 
+LaTeX will take it's own tutorial. However, you don't need to know a lot of LaTeX. I will introduce some rules that will help you write any math equation. Notice that equations are specified in `TexMobject`s instead of `TextMobject`s. 
 
-**LaTex:** When making an equation, the general rule is to put a *$* at the start and end of the text. For example:
 ```python
-text = TextMobject("This is text") #Normal text
-equation = TextMobject("$X$") #This is an equation X
+text = TextMobject("This is text")  # Normal text
+equation = TexMobject("2x+1=3")       # Math equation
 ```
 
-Now for the fun part. In LaTex, you can represent symbols using a backslash and a keyword. THis include theta, alpha, summation, etc. In Manim, it is similar. 
+If you want to put both regular text and math equations in one animation, you can use `$` to at both start and end of the math equations:
+```python
+equation_alternative = TextMobject("Equation: $2x+1=3$") # This is an equation
+```
 
+Now for the fun part. In LaTeX, you can represent symbols using a backslash and a keyword. THis include theta, alpha, summation, integral, fractions, etc.
 ```python
-theta = TextMobject("$\\theta$")
+theta = TexMobject("\\theta")
 ```
-Notice, in Manim, you specify symbols by putting 2 backslash before the keyword. 
+For the sake of escape sequences, we need to use two backslashes to represent `\` in Python
 
 Finally, the I will introduce the syntax for adding subscripts and superscripts. Here is the syntax for superscripts. 
 
 ```python
-superScript_equation = TextMobject("$r^{\\theta}$")
+superScript_equation = TexMobject("r^{\\theta}")
 ```
 
-The ^ symbol signifies superscript. We put the symbol theta as the superscript. Also, when specifying superscript the {} brackets are not displayed in the equation. They help group all the elements you want to add to the superscript. 
+The `^` symbol signifies superscript. We put the symbol theta as the superscript. Also, when specifying superscript the {} brackets are not displayed in the equation. They help group all the elements you want to add to the superscript. 
 
 For subscripts, it is similar. 
 
 ```python
-subScript_equation = TextMobject("$\\theta_{1}$")
```

Comment:
The $ signify an equation. Even if it isn't required its good practice. It should be left there.

Reply:
Keep Original